sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamingQueryWrapper.scala

```diff
@@ -104,4 +104,8 @@ class StreamingQueryWrapper(@transient private val _streamingQuery: StreamExecut
   override def exception: Option[StreamingQueryException] = {
     streamingQuery.exception
   }
+
+  override def getQuerySummary: QuerySummary = {
```

Comment:
So this is the different path with retrieving the status from `recentProgress`, existing one denotes "batches", and new one denotes "overall query".

Reply:
`QuerySummary` has been removed and all these UI data will be tracked in StreamingQueryListener. Mark this resolved.